test/integration/create/test_create.py

```diff
@@ -1068,32 +1109,94 @@ def check_create(tt_cmd, workdir, template, app_name, params, files):
     )
     assert p.returncode == 0
     assert f"Application '{app_name}' created successfully" in p.stdout
-    for f in files:
-        assert os.path.exists(workdir / app_name / f)
+    for f in expected_files:
+        path = workdir / app_name / f
+        assert os.path.exists(path)
+        assert os.access(path, os.W_OK)
 
 
-def get_status_info(tt_cmd, workdir, target):
-    status_cmd = [tt_cmd, "status", target]
+def check_running_cluster(tt_cmd, workdir, app_name, instances, check_func):
+    # Start app.
+    start_cmd = [tt_cmd, "start", app_name]
     p = subprocess.run(
-        status_cmd,
+        start_cmd,
+        cwd=workdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+    )
+
+    try:
+        # Common check.
+        assert p.returncode == 0
+        pid_files = [os.path.join(workdir, app_name, inst, pid_file) for inst in instances]
+        assert wait_files(5, pid_files)
+        assert wait_cluster_started(10, tt_cmd, workdir, app_name, instances)
+
+        # Specific check.
+        check_func(tt_cmd, workdir, app_name)
+
+    finally:
+        # Stop app.
+        stop_cmd = [tt_cmd, "stop", "--yes", app_name]
+        p = subprocess.run(
+            stop_cmd,
+            cwd=workdir,
+            stderr=subprocess.STDOUT,
+            stdout=subprocess.PIPE,
+            text=True,
+        )
+        assert p.returncode == 0
+
+
+def exec_on_inst(tt_cmd, workdir, inst, cmds):
+    return subprocess.run(
+        [tt_cmd, "connect", inst, "-f-"],
         cwd=workdir,
         stderr=subprocess.STDOUT,
         stdout=subprocess.PIPE,
         text=True,
+        input="\n".join(cmds),
     )
-    assert p.returncode == 0
-    return extract_status(p.stdout)
 
 
-def wait_for_master(tt_cmd, workdir, app_name):
-    def has_master():
-        status_info = get_status_info(tt_cmd, workdir, app_name)
-        for key in status_info.keys():
-            if "MODE" in status_info[key] and status_info[key]["MODE"] == "RW":
-                return True
-        return False
+def check_running_config_storage(tt_cmd, workdir, app_name):
+    status_info = get_status_info(tt_cmd, workdir, app_name)
+    master = None
+    replica = None
+    for key in status_info.keys():
+        if status_info[key]["MODE"] == "RW":
+            master = key
+        else:
+            replica = key
+        assert status_info[key]["STATUS"] == "RUNNING"
```

Comment:
```suggestion
    for inst, status in status_info.items():
        if status.get("MODE", "") == "RW":
            master = inst
        else:
            replica = inst
        assert status.get("STATUS", "") == "RUNNING"
```
Just a bit more safety for case if `status` dictionary does not have `STATUS` or `MODE` keys.